exercises/practice/zebra-puzzle/zebra_puzzle.cpp

```diff
@@ -0,0 +1,21 @@
+#include "zebra_puzzle.h"
+
+#include <string>
+
+using namespace std;
```

Comment:
While even some C++ programmers like to have this `using namespace std;`  in their `.cpp` files most C++ experts recommend not to do that. I'd suggest removing that line, we don't want to promote bad practices.